Project.toml

```
@@ -1,6 +1,6 @@
-name = "FDM"
-uuid = "e25cca7e-83ef-51fa-be6c-dfe2a3123128"
-version = "0.6.1"
+name = "FiniteDifferences"
+uuid = "8991a7d4-b31a-11e9-a2a3-2a2ae2dbcce4"
```

```suggestion
uuid = "e25cca7e-83ef-51fa-be6c-dfe2a3123128"
```

Don't change the name